### PR TITLE
make cmake -DDISABLE_QML=1 work again

### PR DESCRIPTION
**Pull request by sethalves**
- make cmake -DDISABLE_QML=1 work again

https://highfidelity.fogbugz.com/f/cases/20046/building-with-cmake-DDISABLE_QML-1-doesn-t-work

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15354/HighFidelity-Beta-PR14465-33378e8.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15354/HighFidelity-Beta-PR14465-33378e8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15354/HighFidelity-Beta-Interface-PR14465-33378e8.exe

**Comment by sethalves** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15355/HighFidelity-Beta-PR14465-33378e8.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15355/HighFidelity-Beta-PR14465-33378e8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15355/HighFidelity-Beta-Interface-PR14465-33378e8.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15356/HighFidelity-Beta-PR14465-c96d8d4.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15356/HighFidelity-Beta-PR14465-c96d8d4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15356/HighFidelity-Beta-Interface-PR14465-c96d8d4.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15358/HighFidelity-Beta-PR14465-36ef355.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15358/HighFidelity-Beta-PR14465-36ef355.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15358/HighFidelity-Beta-Interface-PR14465-36ef355.exe

**Comment by sethalves** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15369/HighFidelity-Beta-PR14465-36ef355.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15369/HighFidelity-Beta-PR14465-36ef355.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15369/HighFidelity-Beta-Interface-PR14465-36ef355.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15377/HighFidelity-Beta-PR14465-f0734a0.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15377/HighFidelity-Beta-PR14465-f0734a0.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15430/HighFidelity-Beta-PR14465-d2a2db1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15430/HighFidelity-Beta-PR14465-d2a2db1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15430/HighFidelity-Beta-Interface-PR14465-d2a2db1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15430/HighFidelity-Beta-PR14465-d2a2db1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15430/HighFidelity-Beta-Interface-PR14465-d2a2db1.exe

**Comment by sethalves** [comment]
to test:
- nothing should change

**Comment by JeffClinton** [comment]
https://highfidelity.manuscript.com/f/cases/20046/building-with-cmake-DDISABLE_QML-1-doesn-t-work

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -4341,9 +4383,13 @@ void Application::mouseDoublePressEvent(QMouseEvent* event) {
 
 void Application::mouseReleaseEvent(QMouseEvent* event) {
 
-    auto offscreenUi = DependencyManager::get<OffscreenUi>();
+#if !defined(DISABLE_QML)
+    auto offscreenUi = getOffscreenUI();
     auto eventPosition = getApplicationCompositor().getMouseEventPosition(event);
     QPointF transformedPos = offscreenUi->mapToVirtualScreen(eventPosition);
+#else
+    QPointF transformedPos;
```
Comment (jherico) [inline interface/src/Application.cpp]: why wouldn't you initialize this to the real mouse position from the event?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15453/HighFidelity-Beta-PR14465-8f21ade.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15453/HighFidelity-Beta-PR14465-8f21ade.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15453/HighFidelity-Beta-Interface-PR14465-8f21ade.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15453/HighFidelity-Beta-PR14465-8f21ade.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15453/HighFidelity-Beta-Interface-PR14465-8f21ade.exe

File: interface/src/Application.cpp (hunk 1 above)
Comment (sethalves) [reply interface/src/Application.cpp]: because it appears to use offscreenUi to make sense of it?

**Comment by JeffClinton** [comment]
Currently testing

**Pull request merged**